mu-plugins/central-hub/tests/phpunit/unit/config-store/_loadConfigFromFilesystem.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ *  Tests for _load_config_from_filesystem
+ *
+ * @package    spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\ConfigStore;
+
+use function KnowTheCode\ConfigStore\_load_config_from_filesystem;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_LoadConfigFromFilesystem
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @group   config-store
+ */
+class Tests_LoadConfigFromFilesystem extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/config-store/internals.php';
+	}
+
+	/**
+	 *  Test should return array when $path_to_file is given.
+	 */
+	public function test_should_return_array_when_path_to_file_is_given() {
+		$path_to_file = CENTRAL_HUB_ROOT_DIR . '/tests/phpunit/fixtures/test-cpt-config.php';
+		$config       = _load_config_from_filesystem( $path_to_file );
+
+		$this->assertArrayHasKey( 0, $config );
```

Comment:
@rgadon107 You want to test against the known fixture configuration to ensure what you get back from `_load_config_from_filesystem` is exactly what you expect.

Here's the code in that function:

https://github.com/rgadon107/cornerstone/blob/7de92a8b0c1a54c3de12a6846b903ba852a272a2/mu-plugins/central-hub/src/config-store/internals.php#L71-L78

Here's what you want to test:

```php
$path_to_file = CENTRAL_HUB_ROOT_DIR . '/tests/phpunit/fixtures/test-cpt-config.php';
$expected = [
     'aaa' => 'bbb',
     'ccc' => 'ddd'
];
$actual       = _load_config_from_filesystem( $path_to_file );

$this->assert( 'foo', $actual[0] );
$this->assertSame( $expected, $actual[1] );
```

1. Test that index 0 is the key.
2. Test that index 1 is the expected configuration parameters array.

Reply:
Tests updated per code review.